webapp/templates/jury/submission_edit_source_modal.html.twig

```diff
@@ -0,0 +1,24 @@
+<div class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">
+                    Submission details
+                </h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            {{ form_start(form) }}
```

Comment:
I assume we still can't use the `form(form))` instead of splitting it out but did you check?